wrapper/ibus/SConstruct

```diff
@@ -71,7 +71,7 @@ PassVariables(envvar, env)
 
 data_dir = env['DATADIR'] + '/ibus-sunpinyin'
 icons_dir = env['DATADIR'] + '/ibus-sunpinyin/icons'
-bin_dir = env['LIBEXECDIR'] + '/ibus-sunpinyin'
+bin_dir = env['LIBEXECDIR'] + '/'
```

Comment:
i still think we should put the executables in `/usr/lib/ibus`.
```suggestion
bin_dir = env['LIBEXECDIR'] + '/ibus'
```

Reply:
also for posterities, https://packages.ubuntu.com/eoan/amd64/ibus-pinyin/filelist, where ibus-pinyin puts its executables like
```
/usr/lib/ibus/ibus-engine-pinyin
/usr/lib/ibus/ibus-setup-pinyin
```

Reply:
also, could you please update the PR using `git push -f` instead creating a new PR to address the review's comments? so we can have a better history of discussion and changes.

Reply:
OK.

Reply:
> also for posterities, https://packages.ubuntu.com/eoan/amd64/ibus-pinyin/filelist, where ibus-pinyin puts its executables like
> 
> ```
> /usr/lib/ibus/ibus-engine-pinyin
> /usr/lib/ibus/ibus-setup-pinyin
> ```

I mean set the default libexecdir as its root, so that we can install ibus-setup-sunpinyin to /usr/lib/ibus/ with the build option --libexecdir=%{_libdir}/ibus or other paths. Changing the default libexecdir in SConstruct is not a good idea.

Reply:
> also, could you please update the PR using `git push -f` instead creating a new PR to address the review's comments? so we can have a better history of discussion and changes.

OK.

Reply:
> > also for posterities, https://packages.ubuntu.com/eoan/amd64/ibus-pinyin/filelist, where ibus-pinyin puts its executables like
> > ```
> > /usr/lib/ibus/ibus-engine-pinyin
> > /usr/lib/ibus/ibus-setup-pinyin
> > ```
> 
> I mean set the default libexecdir as its root, so that we can install ibus-setup-sunpinyin to /usr/lib/ibus/ with the build option --libexecdir=%{_libdir}/ibus or other paths. Changing the default libexecdir in SConstruct is not a good idea.

could you enlighten me why `env['LIBEXECDIR'] + '/ibus-sunpinyin'` is not a good idea?

and i don't think we are "changing" the default libexecdir. actually, we follow the user's will to put the executables *under* the specified libexecdir.